ci/azure-pipelines-docs.yml

```diff
@@ -1,119 +0,0 @@
-# This file configures the `docs` pipeline (https://dev.azure.com/great-expectations/great_expectations/_build)
```

Comment:
We will need to update the branch protections so this pipeline isn't necessary for PR merge.

Reply:
Yes, I think I have the privileges to do this.